modules/default/weather/providers/localWrapper.js

```diff
@@ -0,0 +1,76 @@
+/* global Class */
+
+/* Wrapper to enable the injection of local sensor data */
+const LocalWrapper = Class.extend({
```

Comment:
LocalWrapper doesnt sound very explaining. Can you come up with a better name?
ALso I am not quite clear on how this whole thing works. Maybe add some more context in the jsdoc?

Reply:
Added JSDocs and tried to rename things to be more clear to their use.